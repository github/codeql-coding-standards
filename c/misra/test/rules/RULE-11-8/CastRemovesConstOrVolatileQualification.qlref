rules/RULE-11-8/CastRemovesConstOrVolatileQualification.ql