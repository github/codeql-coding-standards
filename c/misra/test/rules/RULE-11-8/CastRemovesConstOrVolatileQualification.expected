| test.c:4:19:4:33 | (const char *)... | Cast of pointer removes volatile qualification from its base type. |
| test.c:6:13:6:21 | (char *)... | Cast of pointer removes const qualification from its base type. |
